pset-iterators.js

```diff
@@ -114,6 +128,14 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 4:')
 
 // Add your code below this line
+const presidentsBetween1850and1900 = presidents.filter((president) => {
+  //console.log(typeof president.took_office)
+  //let splitDate = parseInt(president.took_office.split(('-')))
+  let splitDate = parseInt(president.took_office.split('-')[0])
```

Comment:
Way to keep data type manipulation into consideration and great use of method chaining to manipulate a string here.